### DISCUSSION TITLE
Restrict directus to localhost

### DISCUSSION THREAD

**Discussion by 720720** [directus/directus#10629] [Feature Requests]
Im running directus behind a nginx reverse proxy for ssl termination.
Currently I have to use a nftables firewall to block HTTP APP and API access.
It would be great to have a configuration option to restrict directus HTTP APP and API to localhost.

**Comment by joselcvarela** [comment]
Hello @720720 
Yeah it seems the app is listening on all interfaces. This can be easily added here https://github.com/directus/directus/blob/9105c1e42c0dbf125579b27ac173d881b96db17f/api/src/server.ts#L129

Just add `host` as second parameter where `const host = env.HOST;`.
Then on your side, you can specify `HOST=127.0.0.1` on your `.env`

Do you want to open a PR for it?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
